core/src/test/scala/zio/config/testsupport/MapConfigTestSupport.scala

```diff
@@ -40,27 +40,30 @@ object MapConfigTestSupport {
 
     object AwsConfig {
       val description: ConfigDescriptor[String, String, AwsConfig] =
-        nested("aws")(
-          (string("key") |@| string("secret") |@| KinesisConfig.description)(AwsConfig.apply, AwsConfig.unapply)
+        first("aws")(
```

Comment:
Why is there a need to change all these to `first`? They weren't lists before, right?

Reply:
`head("a")(string)` accepts both `a: "v"` `a: ["v1", "v2"]` as input. There is no need for list in description for `head`. There should be a list in configs and this test generates lists in place of scalars.